README.md

```diff
@@ -34,8 +34,8 @@ If you would like to contribute, please fork the repository, introduce your chan
 
 ### License
 
-The code in this repository is licensed under the terms of the the Apache License 2.0.
-
+- The code in this repository is licensed under the terms of the the Apache License 2.0.
+- With the exception of `bin/cliain/src/transcode/*` directory which contains code licensed under GPL v3.0
```

Comment:
I have no idea whether this is sufficient so I would like to mention @timorl (?) to judge

Reply:
That is sth I noticed is done for [Substrate itself](https://github.com/paritytech/substrate#license). I do not know how this is supposed to work in practice, this is well over my head.

Reply:
Huh, I must have missed this, and I am no expert either, but I'm pretty sure we should also include the text of the GPL in a LICENSE-like file (as Substrate does and as the GPL explicitly requires). In practice this should be forwarded to some lawyers to ensure it's ok.

Reply:
resolved, code is a dependency now